cmd/crates/stellar-ledger/src/lib.rs

```diff
@@ -0,0 +1,765 @@
+mod transport_zemu_http;
```

Comment:
This file is where the ADPU commands are created and sent to the ledger device (or emulator). It is based on this js implementation: https://github.com/LedgerHQ/ledger-live/blob/882c77d74d7812116d6820e3f76f74bfa67a609b/libs/ledgerjs/packages/hw-app-str/src/Str.ts

The repo of the stellar application that is installed on the ledger device itself was also a handy reference when determining how to generate these commands: https://github.com/LedgerHQ/app-stellar/tree/develop/docs

There is still work to be done on error handling, and some additional cleanup. Feedback is definitely welcome!